src/pages/Login/Login.jsx

```diff
@@ -0,0 +1,73 @@
+import {Link, useNavigate} from "react-router-dom";
+import "./Login.css";
+import {useState} from "react"
+import axios from "axios";
+import {useAuth} from "../../Context/authContext";
+
+
+export function Login() {
+
+    const {setUser, setToken} = useAuth()
+    const navigate = useNavigate();
+    const [loginForm, setLoginForm] = useState({email: "", password: ""})
+
+    const loginHandler = async (e)  => {
+        e.preventDefault();
+        try {
+            const {status, data: {foundUser, encodedToken}} = await axios.post(`/api/auth/login`, loginForm
```

Comment:
Liked the way, you destructured ```foundUser``` and ```encodedToken``` but be careful when they're not present in the data. In such cases, it might break you authentication logic.